src/pages/deriv-go/_footer-banner.js

```diff
@@ -45,7 +46,18 @@ const BackgroundWrapper = styled(Flex)`
         flex-direction: column-reverse;
     }
 `
-
+const ButtonDerivGO = styled(Button)`
+    opacity: 0;
+    @media ${device.tabletL} {
```

Comment:
why are we making it `opacity: 0` ? Can we use `Mobile` component to only show the button on mobile?

Reply:
Had some alignment issues when I was using Desktop and Mobile components but with media query it worked.

Reply:
Fixed the issue because with opacity I can still click the button even though it doesn't appear. Thanks 👍